src/CommunityToolkit/Xamarin.CommunityToolkit/Behaviors/Animations/AnimationBehavior.shared.cs

```diff
@@ -38,21 +54,29 @@ protected override void OnDetachingFrom(VisualElement bindable)
 		}
 
 		protected override async void OnTriggerHandled(object? sender = null, object? eventArgs = null)
+		{
+			await OnAnimate();
+
+			base.OnTriggerHandled(sender, eventArgs);
+		}
+
+		static object CreateAnimateCommand(BindableObject bindable)
+			=> new AsyncCommand(((AnimationBehavior)bindable).OnAnimate);
+
+		async Task OnAnimate()
 		{
 			if (isAnimating)
 				return;
 
 			isAnimating = true;
 
 			if (AnimationType != null)
-				await AnimationType.Animate((View?)sender);
+				await AnimationType.Animate((View?)View);
 
 			if (Command?.CanExecute(CommandParameter) ?? false)
```

Comment:
I think this behavior is still correct but wanted to check it through with someone. Basically if the `AnimateCommand` is fired should the `Command` on the `AnimationBehavior` also fire?

Reply:
1) I think no because this command is called when EVENT is raised
But good news
2) Don't worry about these lines. They don't exist anymore since https://github.com/xamarin/XamarinCommunityToolkit/pull/1162 (in main branch)

Reply:
OK I have left this thanks